Sources/Player/OfflineEngine/Internal/Storage/GRDBStorage/GRDBOfflineStorage.swift

```diff
@@ -122,7 +117,22 @@ private extension GRDBOfflineStorage {
 	static func databaseURL() throws -> URL {
 		let appSupportURL = PlayerWorld.fileManagerClient.applicationSupportDirectory()
 		let directoryURL = appSupportURL.appendingPathComponent("PlayerOfflineDatabase", isDirectory: true)
-		try PlayerWorld.fileManagerClient.createDirectory(at: directoryURL, withIntermediateDirectories: true)
+		if PlayerWorld.fileManagerClient.fileExists(atPath: directoryURL.path, isDirectory: nil) {
+			var attributes = try? PlayerWorld.fileManagerClient.attributesOfItem(directoryURL.path)
+			if attributes != nil,
```

Comment:
`if let attributes` maybe? Then you don't need to force unwrap them later

Reply:
I need to modify it, so it needs to be a `var` 🤔 

I tried a few different ways, and I did not like any of them...

Reply:
If I add the log, I can do `try` instead of `try?`, and it improves the readability of the code a lot.

Reply:
done!